specification/communication/data-plane/Chat/stable/2021-09-07/communicationserviceschat.json

```diff
@@ -1688,6 +1700,38 @@
           "example": "Bob Admin"
         }
       }
+    },
+    "DeletionStrategy": {
```

Comment:
In other azure services a common term to use is "retention". What are your thoughts on having something like:
```json
"retentionPolicy": {
    "kind": "basedOnThreadCreationDate",
    "daysAfterCreation": 90
}
```